### ISSUE TITLE
Implement Nick Name Table for DRC Data Pulls (BlueJeans 11.18.19)

### ISSUE THREAD

**Issue by qkyho** [NUARIG/all-of-us-helper#145]
Implement nick name table for DRC data pulls to account in common nick names for mismatch lists

**Comment by qkyho** [comment]
Per SOP found here: https://docs.google.com/document/d/15zjSUEF5USpfoQu0TvAevyIIMYpPeTDX3fvDCqSHcGM/edit#heading=h.j97spx63jtdr

Ignore, special character differences, nicknames, suffixes, prefixes, hyphens, and character cases for DRC data pulls.

**Comment by ybushmanova** [comment]
@OSNorthwestern is this still relevant?

**Comment by OSNorthwestern** [comment]
@ybushmanova Yes still relevant. Basically being able to pull from aliases table in EPIC when comparing OMOP name values and HealthPro values would save lot of time.

**Comment by NiviyaGeorge** [comment]
@ybushmanova, are we able to implement this? Basically we are looking to include nicknames in the mismatch process prior to EHR submission. Nicknames/Aliases are what majority of those mismatch list consists of. If DRC data pull can include nicknames/Aliases table in Epic, that would short the list.

**Comment by ybushmanova** [comment]
@NiviyaGeorge I'll check with Mike on this and let you know

**Comment by pshum1** [comment]
@NiviyaGeorge we were looking into this ticket and wanted to know where you would want to see the nicknames? Would it be a column in the mismatch table prior to submission?

**Comment by NiviyaGeorge** [comment]
@pshum1, Yes definitely before the submission. I believe we were trying to see the Excel sheet that you provided me to check for mismatch should consider nicknames as well. The step when doing the comparison between OMOP name values and HealthPro values and need to include the nick name values as well.
